imm.yaml

```diff
@@ -0,0 +1,120 @@
+﻿-
+    Name: 'Shared Engineering Standards'
+    Id: 74e29f9b-6dca-4161-8fdd-b468a1eb185d
+    Measures:
+        -
+            Score: 0
+            Description: None
+-
+    Name: 'Coding Standards'
+    Id: f6f6490f-9493-4dc3-a674-15584fa951d8
+    Measures:
+        -
+            Score: 0
+            Description: None
+-
+    Name: 'Executable Specifications'
+    Id: bb49fb94-6ab5-40c3-a6da-dfd2e9bc4b00
+    Measures:
+        -
+            Score: 0
```

Comment:
Is stuff untested, or have you just not updated this?

Reply:
I wasn't sure what our position on IMMs for new projects are. I've added a card for an initial IMM review to the project board; I think we should do it as a separate PR.